src/base/emu-i386/simx86/sigsegv.c

```diff
@@ -181,7 +181,8 @@ int e_vgaemu_fault(sigcontext_t *scp, unsigned page_fault)
       }
       return 1;
     }
-    else if (page_fault >= 0xc0 && page_fault < (0xc0 + vgaemu_bios.pages)) {	/* ROM area */
+    else if((page_fault >= 0xc0 && page_fault < (0xc0 + vgaemu_bios.pages)) ||
+	    (!config.umb_f0 && page_fault >= 0xf0 && page_fault < 0xf4)) {  /* ROM area */
```

Comment:
Please use DOSEMU_LMHEAP_OFF>>4 instead
of hard-coding segments.

Reply:
Or maybe use memcheck API?
Please see what suits.
memcheck seems under-used.

Reply:
tried that in #1972, then this patch can be adapted to that.